### ISSUE TITLE
Internal Server Error: Array to string conversion

### ISSUE THREAD

**Issue by RobErskine** [FoxyCart/foxycart-craft#5]
When devmode is turned off, we're getting an "Internal Server Error: Array to string conversion" error when visiting the foxycart section of the Craft admin center. 

When devmode is turned on, the page loads as intended but instead (of course) shows test data. I suspect that the data coming back from `craft.foxyCart.listTransactions` may have changed? I'm not sure what else to check. If there is any other detail I can provide please let me know.

**Comment by adamjudd** [comment]
@RobErskine,

Thanks for the report! To confirm - you're seeing that error when just visiting the FoxyCart admin page that lists transactions, not on the settings page for the plugin?

Could we confirm your Craft version too please?

We'll dig in and see if we can spot the issue.

**Comment by RobErskine** [comment]
Heya @adamjudd,

Yeah this is occuring when you hit /{adminURL}/foxycart.

![foxycart error](https://cl.ly/df460cf4c297/Screen%20Shot%202019-01-29%20at%2011.47.36%20AM.jpg)

And turning on dev mode doesn't hint at any other error messages, because instead it just shows test data.

Lastly we were on Craft v 2.6.2985 when we first noticed this issue. I just upgraded us to Craft v 2.7.4 and it is still happening.
